yoyodyne/models/pointer_generator.py

```diff
@@ -501,10 +508,10 @@ def forward(self, batch: batches.PaddedBatch) -> torch.Tensor:
         """
         assert batch.has_features
         source_encoded, (h_source, c_source) = self.encode(
-            batch.source, self.encoder
+            batch.source, self.source_embeddings, self.encoder
         )
         features_encoded, (h_features, c_features) = self.encode(
-            batch.features, self.feature_encoder
+            batch.features, self.feature_embeddings, self.feature_encoder
```

Comment:
Question: Did batch.features previously index the embeddings matrix incorrectly? Do you have any idea when this was introduced?

Reply:
I remember indexing behavior being correct prior to the `indexes` implementation. So it may have been introduced during that update.

Reply:
Ah, I think I know why this behavior wasn't affected. Since `pointer_generator` just uses one embedding and was instantiating from vocabulary, it was already the size of the conjoined `source + feature` space.  Since `batch.features` was still indexed in order with source vocabulary, it would have had the proper encodings.